jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/LambdaConfigPanel.form

```diff
@@ -146,14 +146,17 @@
         <constraints>
           <grid row="0" column="1" row-span="1" col-span="2" vsize-policy="3" hsize-policy="3" anchor="0" fill="3" indent="0" use-parent-layout="false"/>
         </constraints>
-        <properties/>
+        <properties>
```

Comment:
This is also used in CreateFunction, fyi. Make sure you didnt break that

Reply:
Removing this change from this PR since reindexing is taking > 30 min, we can tackle this in a different PR